src/Models/Tracers.jl

```diff
@@ -3,7 +3,7 @@ module Tracers
 using NamedArrays
 
 export typical_detritus,
-    typical_nutrients, simplified_phytoplankton_growth, simplified_zooplankton_growth
+    typical_nutrients, single_nutrient_phytoplankton_growth, simplified_zooplankton_growth
```

Comment:
Should this be changed to be consistent with naming conventions elsewhere?

```suggestion
    typical_nutrients, phytoplankton_growth_single_nutrient, zooplankton_growth_simplified
```